packages/devtools_app/lib/src/screens/debugger/program_explorer.dart

```diff
@@ -188,7 +188,7 @@ class _ProgramExplorerRow extends StatelessWidget {
         buffer.write(closingTag);
       }
     }
-    buffer.write(') → ');
+    buffer.write(') -> ');
```

Comment:
You can conditionally only show `->` on the web with the `kIsWeb` check (see example: https://github.com/flutter/devtools/pull/3353/files)

Also, could you add a TODO to change back to the unicode arrow once https://github.com/flutter/flutter/issues/102518 is fixed?

Reply:
SGTM! I'll make it happen :D